src/platforms/javascript/common/performance/capturing/automatic.mdx

```diff
@@ -1,19 +1,28 @@
 ---
-title: Automatic Capture
-sidebar_order: 20
-description: "Learn how to configure what performance data is automatically captured."
+title: Add Automatic Instrumentation
+sidebar_order: 1
+description: "Learn how to add automatic instrumentation to captured transactions."
 ---
 
-The `@sentry/tracing` package provides a `BrowserTracing` integration to add automatic instrumentation for monitoring the performance of browser applications. By default, this integration will create a new transaction for each page load and navigation event, and will create a child span for every `XMLHttpRequest` or `fetch` request which occurs while those transactions are open.
+<Note>
+
+To _automatically_ capture transactions, you must first <PlatformLink to="/performance/">enable tracing in your app.</PlatformLink>
+
+</Note>
+
+The `@sentry/tracing` package provides a `BrowserTracing` integration to add _automatic_ instrumentation for monitoring the performance of browser applications.
+
+## What Automatic Instrumentation Provides
+
+The `BrowserTracing` integration creates a new transaction for each page load and navigation event, and creates a child span for every `XMLHttpRequest` or `fetch` request that occurs while those transactions are open. Learn more about [traces, transactions, and spans](/product/performance/distributed-tracing/).
+
+## Enable Automatic Instrumentation
 
 To enable this automatic tracing, include the `BrowserTracing` integration in your SDK configuration options. (Note that when using ESM modules, the main `@sentry/*` import must come before the `@sentry/tracing` import.)
 
-```javascript {tabTitle: ESM}
-// If you're using one of our integration packages, like `@sentry/react` or `@sentry/angular`,
-// substitute its name for `@sentry/browser` here
-import * as Sentry from "@sentry/browser";
-import { Integrations as TracingIntegrations } from "@sentry/tracing"; // Must import second
+After configuration, you will see both `pageload` and `navigation` when viewing transactions in sentry.io.
 
+```javascript {tabTitle: ESM}
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
```

Comment:
We dropped the imports in the new code block which gets a little confusing with the stence above which mentions the order.

```suggestion
```javascript {tabTitle: ESM}
// If you're using one of our integration packages, like `@sentry/react` or `@sentry/angular`,
// substitute its name for `@sentry/browser` here
import * as Sentry from "@sentry/browser";
import { Integrations as TracingIntegrations } from "@sentry/tracing"; // Must import second

Sentry.init({
  dsn: "___PUBLIC_DSN___",
```